x-pack/plugins/security/public/authentication/logged_out/logged_out_page.tsx

```diff
@@ -18,9 +24,11 @@ import { AuthenticationStatePage } from '../components';
 
 interface Props {
   basePath: IBasePath;
+  customBranding: CustomBrandingStart;
 }
 
-export function LoggedOutPage({ basePath }: Props) {
+export function LoggedOutPage({ basePath, customBranding }: Props) {
+  const customBrandingValue = useObservable(customBranding.customBranding$);
```

Comment:
Question: Is there a reason to do `useObservable(customBranding.customBranding$)` vs `useObservable(customBranding.hasCustomBranding$)` ?

Reply:
`hasCustomBranding$` resolves to a boolean value that is true if any one of the custom branding properties is set. `customBranding$` resolves to an object whose properties are custom branding values.